mail_activity_reminder/models/mail_activity.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from pytz import timezone, UTC
+
+from odoo import _, api, fields, models
+
+
+class MailActivity(models.Model):
+    _inherit = 'mail.activity'
+
+    last_reminder = fields.Date()
+
+    @api.model
+    def _get_activities_to_remind(self):
+        utc_now = self.env.context.get(
+            'mail_activity_reminder_now',
+            fields.Datetime.now()
+        ).replace(tzinfo=UTC)
+
+        activities_to_remind = self.env['mail.activity']
+        for activity in self.search([('active', '=', True)]):
```

Comment:
Module should depend on mail_activity_done since activity is not a base field?

Reply:
Sorry, but I don't get you. What field do you refer to?

Reply:
`active` is very much field of base model https://github.com/odoo/odoo/blob/2ad6cb4feb01dbb394ab50ce9f8c1fd6da30bf8c/addons/mail/models/mail_activity.py#L39

Reply:
And I'm not accessing any field named `activity`

Reply:
Weird I got an error on it while installing, let me check again.

Reply:
A stacktrace would be helpful

Reply:
```
  File "/Users/brahoo/PycharmProjects/odoo/community/mail_activity_reminder/models/mail_activity.py", line 47, in _process_reminders
    activities = self._get_activities_to_remind()
  File "/Users/brahoo/PycharmProjects/odoo/community/mail_activity_reminder/models/mail_activity.py", line 22, in _get_activities_to_remind
    for activity in self.search([('active', '=', True)]):
  File "/Users/brahoo/PycharmProjects/odoo/odoo-12.0/odoo/models.py", line 1579, in search
    res = self._search(args, offset=offset, limit=limit, order=order, count=count)
  File "/Users/brahoo/PycharmProjects/odoo/odoo-12.0/odoo/models.py", line 4127, in _search
    query = self._where_calc(args)
  File "/Users/brahoo/PycharmProjects/odoo/odoo-12.0/odoo/models.py", line 3919, in _where_calc
    e = expression.expression(domain, self)
  File "/Users/brahoo/PycharmProjects/odoo/odoo-12.0/odoo/osv/expression.py", line 673, in __init__
    self.parse()
  File "/Users/brahoo/PycharmProjects/odoo/odoo-12.0/odoo/osv/expression.py", line 854, in parse
    raise ValueError("Invalid field %r in leaf %r" % (left, str(leaf)))
```

Reply:
complains about `active` field?

Reply:
What I can read from the traceback yes. But seems to working from time to time too. The above traceback I got in a clean DB with demo data loaded. Only installed this new module.

Reply:
No need to add `('active', '=', True)`, as that's the default behavior.

@CasVissers if you are not able to reproduce it on runbot, then it can be due to outdated code or any bad interaction

Reply:
Consider closed.

Reply:
Updated the code to not mention `active`